namer/types.py

```diff
@@ -366,7 +366,19 @@ def __init__(self):
             self.set_uid = os.getuid()
             self.set_gid = os.getgid()
 
-    def __dict__(self):
+    def __str__(self):
+        config = self.to_dict()
+
+        output = []
+        for key in config:
+            output.append(f"{key}:")
+            for value in config[key]:
+                output.append(f"  {value}: {config[key][value]}")
+
+        # return json.dumps(config, indent=2)
+        return '\n'.join(output)
+
+    def to_dict(self) -> dict:
         porndb_token = "None In Set, Go to https://metadatapi.net/ to get one!"
         if self.porndb_token is not None:
             porndb_token = "*" * len(self.porndb_token)
```

Comment:
wont this result in a bug where any edit to other values will update the token to a bunch of asterisks?